src/Toolkit/V.php

```diff
@@ -449,7 +449,7 @@ public static function __callStatic(string $method, array $arguments): bool
 	 * Checks if the value matches the given regular expression
 	 */
 	'match' => function ($value, string $pattern): bool {
-		return preg_match($pattern, $value) !== 0;
+		return preg_match($pattern, (string)$value) !== 0;
```

Comment:
Shouldn't this actually be:

```suggestion
		return preg_match($pattern, (string)$value) === 1;
```

Otherwise the validator will return `true` if `preg_match()` returns `false` (= if an error occurred).

Reply:
I think you're right. Didn't question it since I hadn't touched that part, but especially with strict comparisons `false` would be an issue now.